synapseclient/models/virtualtable.py

```diff
@@ -536,8 +536,19 @@ async def main():
             ```
         """
         # Check for unsupported operations in defining_sql
+
         if self.defining_sql:
-            if any(clause in self.defining_sql.upper() for clause in ["JOIN", "UNION"]):
+            import re
```

Comment:
I don't think this is harmful to be imported here, but this could be moved to import at the top of the module